docs/src/app/components/pages/components/lists.jsx

```diff
@@ -34,11 +36,40 @@ const { Colors } = Styles;
 const Code = require('lists-code');
 const CodeExample = require('../../code-example/code-example');
 const CodeBlock = require('../../code-example/code-block');
+let SelectableList = SelectableContainerEnhance(List);
+
+function wrapState(ComposedComponent) {
```

Comment:
Is there any way we could avoid having this helper function to wrap `SelectableList`? And do it straight using props in the render method of `ListsPage`? I think it would be a good idea to keep the docs code as straightforward as possible..